index.d.ts

```diff
@@ -7,6 +7,15 @@ declare namespace debounceFn {
 		*/
 		readonly wait?: number;
 
+		/**
+		Maximum time to wait until the `input` function is called.
+		Only applies when after is true.
```

Comment:
```suggestion
		Only applies when the `after` option is `true`.
```

Reply:
Why only when `after` is `true`?

Reply:
I wasn't sure how it should behave if `before` is `true`, but it does work fine with a very small change